### DISCUSSION TITLE
How strapi search for a json field?

### DISCUSSION THREAD

**Discussion by casulit** [strapi/strapi#7743] [Legacy Topics]
Does strapi support search (_q) in json fields?

**Comment by MattieBelt** [comment]
Hey @casulit,

No search on a JSON field isn't implemented, but someone offered [a solution on SO](https://stackoverflow.com/questions/59232964/is-there-a-way-to-filter-json-data-format-field-in-strapi).
Also checkout all the possible [(filter) parameters](https://strapi.io/documentation/v3.x/content-api/parameters.html) for REST.

Duplicate of #7353
